x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/amazonbedrock/completion/AmazonBedrockCompletionTaskSettings.java

```diff
@@ -30,17 +30,17 @@
 import static org.elasticsearch.xpack.inference.services.amazonbedrock.AmazonBedrockConstants.TOP_K_FIELD;
 import static org.elasticsearch.xpack.inference.services.amazonbedrock.AmazonBedrockConstants.TOP_P_FIELD;
 
-public class AmazonBedrockChatCompletionTaskSettings implements TaskSettings {
+public class AmazonBedrockCompletionTaskSettings implements TaskSettings {
     public static final String NAME = "amazon_bedrock_chat_completion_task_settings";
 
-    public static final AmazonBedrockChatCompletionRequestTaskSettings EMPTY_SETTINGS = new AmazonBedrockChatCompletionRequestTaskSettings(
+    public static final AmazonBedrockCompletionRequestTaskSettings EMPTY_SETTINGS = new AmazonBedrockCompletionRequestTaskSettings(
         null,
         null,
         null,
         null
     );
 
-    public static AmazonBedrockChatCompletionTaskSettings fromMap(Map<String, Object> settings) {
+    public static AmazonBedrockCompletionTaskSettings fromMap(Map<String, Object> settings) {
```

Comment:
If we had a constant empty `AmazonBedrockCompletionTaskSettings` we could return that here if `settings` was empty and avoid some work.